config.default.yaml

```diff
@@ -128,6 +129,7 @@ lines:
     220.: "Al/St 240/40 2-bundle 220.0"
     300.: "Al/St 240/40 3-bundle 300.0"
     380.: "Al/St 240/40 4-bundle 380.0"
+  dc_type: "DC_custom_linetype"
```

Comment:
Also here, please, add a description and options for the values

Reply:
Done!